graphs/dsl/src/main/kotlin/com/anaplan/engineering/azuki/graphs/dsl/declaration/UndirectedGraphDeclarations.kt

```diff
@@ -0,0 +1,20 @@
+package com.anaplan.engineering.azuki.graphs.dsl.declaration
+
+import com.anaplan.engineering.azuki.graphs.dsl.DirectedGraphBlock
+import com.anaplan.engineering.azuki.graphs.dsl.UndirectedGraphBlock
+
+interface UndirectedGraphDeclarations {
+
+    fun thereIsAGraph(graphName: String)
+
+    fun thereIsAGraph(graphName: String, init: UndirectedGraphBlock.() -> Unit)
```

Comment:
🚫 As per https://github.com/anaplan-engineering/azuki/pull/40#discussion_r1576080467, these should be called `thereIsAnUndirectedGraph`. In `GraphGiven` you can then have:
```
fun thereIsAGraph(graphName: String) = thereIsAnUndirectedGraph(graphName)
override fun thereIsAnUndirectedGraph(graphName: String) { ... }
```
etc.